submissions/goldOchim/aboutMe/HTML/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>
+            gold-ecommerce
+        </title>
+    </head>
+    <body>
+        <h1>
+            GA-JETS STORES 
+        </h1>
+           
+            <table>
+               
+                <tr>
+                    <th>  <img src="headp5.jpeg">
+                    </img> </th>
```

Comment:
`</img>` is not used properly

Reply:
Is it the bad coding practice of where it is placed or something else. I don't really understand.

Reply:
img tag is a self-closing tag. It does not have a closing tag

Reply:
Oh, okay. I have made the corrections and attached the images.